frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -0,0 +1,303 @@
+
+  return (
+    <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData
+        className={blockPoolColumnInfo.name.classes}
+        columnID={blockPoolColumnInfo.name.id}
+      >
+        <ResourceLink kind={referenceFor(obj)} name={name} namespace={namespace} />
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.status.classes}
+        columnID={blockPoolColumnInfo.status.id}
+      >
+        <Status status={phase} />
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.storageclasses.classes}
+        columnID={blockPoolColumnInfo.storageclasses.id}
+      >
+        <PopoverHelper names={poolScNames} text="StorageClasses" kind={StorageClassModel} />
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.replicas.classes}
+        columnID={blockPoolColumnInfo.replicas.id}
+      >
+        {replica}
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.usedcapacity.classes}
+        columnID={blockPoolColumnInfo.usedcapacity.id}
+      >
+        {rawCapacity}
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.mirroringstatus.classes}
+        columnID={blockPoolColumnInfo.mirroringstatus.id}
+      >
+        {mirroringStatus ? t('ceph-storage-plugin~Enabled') : t('ceph-storage-plugin~Disabled')}
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.overallImagehealth.classes}
+        columnID={blockPoolColumnInfo.overallImagehealth.id}
+      >
+        <Tooltip content={`${t('ceph-storage-plugin~Last Synced')} ${formatedDateTime}`}>
+          <StatusIconAndText
+            title={mirroringImageHealth}
+            icon={healthStateMapping[mirroringImageHealth]?.icon}
+          />
+        </Tooltip>
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.compressionstatus.classes}
+        columnID={blockPoolColumnInfo.compressionstatus.id}
+      >
+        {compressionStatus ? t('ceph-storage-plugin~Enabled') : t('ceph-storage-plugin~Disabled')}
+      </TableData>
+      <TableData
+        className={blockPoolColumnInfo.compressionsavings.classes}
+        columnID={blockPoolColumnInfo.compressionsavings.id}
+      >
+        {'-'}
+      </TableData>
+      <TableData className={Kebab.columnClass}>
+        <ResourceKebab
+          actions={menuActions}
+          kind={referenceFor(obj)}
+          resource={obj}
+          isDisabled={
+            obj?.metadata?.deletionTimestamp ||
+            props.cephCluster?.metadata?.name === CEPH_EXTERNAL_CR_NAME ||
+            isDefaultPool(obj)
+          }
+        />
+      </TableData>
+    </TableRow>
+  );
+};
+
+const BlockPoolList: React.FC<BlockPoolListProps> = (props) => {
+  const { t } = useTranslation();
+
+  // Hooks
+  const [storageClasses, setStorageClasses] = React.useState<OcsStorageClassKind[]>([]);
```

Comment:
```suggestion
  const [storageClasses, setStorageClasses] = React.useState<StorageClassKind[]>([]);
```
Why not use the generic Storage Class Kind?

Reply:
But why we need a default one when we have our own flavor of storage class kind, The purpose of OcsStorageClassKind  to read and write our own parameters like pool and KMS  related parameters.